packages/plugins/Scale/src/store/index.ts

```diff
@@ -70,12 +66,45 @@ export const makeStoreModule = () => {
             : `${scaleNumber} m`
         commit('setScaleWithUnit', newValue)
       },
+      zoomToScale(
+        { dispatch, rootGetters: { configuration } },
+        zoomLevel: number
+      ): void {
+        if (configuration?.scale?.zoomMethod) {
+          dispatch(configuration.scale.zoomMethod, zoomLevel, { root: true })
```

Comment:
```suggestion
      zoomToScale({ dispatch, getters }, zoomLevel: number): void {
        const { zoomMethod } = getters
        if (zoomMethod.length > 0) {
          dispatch(zoomMethod, zoomLevel, { root: true })
```
We can simply use the local getter if we have it :D